<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected Account :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
    <widget class="QLabel" name="WodaFolder_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Woda Folder :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="WodaFolder_input">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_button">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse..</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetNotificationPosition" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline;&quot;&gt;Notification Positon :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonBottomLeft">
        <property name="text">
         <string>Bottom Left</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupNotification</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButtonBottomRight">
        <property name="text">
         <string>Bottom Right</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupNotification</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioButtonTopRight">
        <property name="text">
         <string>Top Right</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupNotification</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="radioButtonTopLeft">
        <property name="text">
         <string>Top Left</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupNotification</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Connexion">
    <attribute name="title">
     <string>Connexion</string>
    </attribute>
    <widget class="QCheckBox" name="checklimitdl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Limit download speed</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Limit to :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_limitdl">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>42</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>kb/s</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checklimitup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Limit upload speed</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Limit to :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>kb/s</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_limitup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupNotification"/>
 </buttongroups>
</ui>
